__tests__/app.test.js

```diff
@@ -126,3 +132,26 @@ describe("GET /api/articles/:article_id/comments", () => {
     }
   });
 });
+describe("Post /api/articles/:article_id/comments", () => {
+  test("posts a comment to an article", async () => {
+    const response = await request(app).post("/api/articles/1/comments").send({
+      username: "butter_bridge",
+      body: "This is a test comment",
+    });
+    expect(response.status).toBe(201);
+    const { comment } = response.body;
+    expect(comment).toHaveProperty("comment_id");
+    expect(comment).toHaveProperty("author", "butter_bridge");
```

Comment:
great we can check for more properties here and just have a type for their value - eg comment id would be number